mautic/mautic
# [UI] Update placeholders and plugins logos

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #13147 



#### Description:

Based on feedback:

> Imagery: replace the placeholder images by more modern ones

The user is suggesting that the current placeholder images used in the platform appear outdated and should be replaced with more contemporary visuals. This feedback indicates that the imagery does not align with modern design trends or the user’s aesthetic expectations.

The user’s broader need is for a visually appealing platform that reflects current design standards and resonates with a modern audience. 

To address this feedback, this PR updates Mautic placeholder images with modern, high-quality images and logos that better represent current brands.

![image](https://github.com/mautic/mautic/assets/116097999/ea958d53-8522-4699-9fa5-e08ba214dd58)

![image](https://github.com/mautic/mautic/assets/116097999/1128243b-d55c-4fab-a434-30d583069015)

- [x] fix .img-responsive with to fill 100%



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts > create new > See the new placeholder for contacts without profile pic
3. Open right sidebar > Plugins > Look at the new social and brands logos
